modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCenter.java

```diff
@@ -131,6 +131,9 @@ public String getCredentialOpenApiUrl(Csp csp, CredentialType type) {
      * @return the credentials.
      */
     public List<AbstractCredentialInfo> getCredentialsByUser(String xpanseUser) {
+        if (StringUtils.isBlank(xpanseUser)) {
```

Comment:
Hi @jinyangyang222 
in the getCredentialsByUser  API method -> we can make user parameter as "required".
And here instead of returning null, we must return empty list.